### DISCUSSION TITLE
Broker Load Hdfs 10亿量级数据失败

### DISCUSSION THREAD

**Discussion by 666chaoge666** [apache/doris#8598] [A - General / Q&A] [answered]
Broker Load 导入Hdfs 10亿量级数据失败
做了以下尝试，没有改善：
1 增加以及减少单个 tablet 最大 version 的数量：max_tablet_version_num
2 增加broker节点导入数据上限：max_bytes_per_broker_scanner
3 增加导入任务的内存：exec_mem_limit
4 修改compaction线程：max_compaction_threads
5 修改BE 上单个 Tablet 的写入速度限制：push_write_mbytes_per_sec

！！！跪求大神帮忙看下！！！

以下为报错信息：
W0321 17:35:01.568506 12540 tablet_sink.cpp:133] index channel has intolerable failure. 192.168.12.218, err: add row failed. NodeChannel[16764-10005], load_id=1906b7af29354e8a-beb34b1b8954ebe8, txn_id=4089, node=10005:8060, add batch req success but status isn't ok, err: tablet writer write failed, tablet_id=16817, txn_id=4089, err=-238;

**Comment by dataalive** [comment, marked as answer]
The -238 error usually occurs when the amount of imported data in the same batch is too large, which leads to too many Segment files for a certain tablet (the default is 200, which is controlled by the BE parameter max_segment_num_per_rowset). At this time, it is recommended to reduce the amount of data imported in one batch, or to appropriately increase the value of the BE configuration parameter to solve the problem.

[#](https://doris.apache.org/faq/error.html#e4-tablet-110309738-has-few-replicas-1-alive-backends-10003)

does it help ?

**Reply by 666chaoge666** [reply]
尝试了增大参数max_segment_num_per_rowset，可以解决当前失败任务问题，后续持续关注，感谢大神帮助
